### PR TITLE
Update policies in example article

### PR DESCRIPTION
**Pull request by katjuell**
### What should this PR do?

We're updating the policies from the example article on the UI to keep everything as up-to-date as possible, so I wanted to port those changes over here as well! Once https://github.com/chainguard-dev/console-ui/pull/574 gets merged in, I will mark this as ready for review, in case there are other changes forthcoming.

### Why are we making this change?

Trying to keep our example, public-facing policies up-to-date and consistent across our properties!

### CODE REVIEW

**Review by SharpRake — APPROVED** [review]
I fixed one minor typo that was unrelated to your changes (guessing it had been in this guide for a while) but your changes all look good to me!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md [hunk 1]
```diff
@@ -145,70 +142,123 @@ metadata:
...
+        identities:
+          # Matches the Github Actions OIDC issuer
+          - issuer: https://token.actions.githubusercontent.com
+            # Matches a specific GitHub workflow on main branch. Here we use the
+            # Sigstore policy controller example testing workflow as an example.
+            subject: "https://github.com/sigstore/policy-controller/.github/workflows/release.yaml@refs/tags/v0.3.0"
 ```
 
-## Policy allowing all images
+## Policy allowing GKE images
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md:159]: Should this include "trusted" in the heading?

**Review by ltagliaferri — APPROVED** [review]
Thank you, Katy!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md (hunk 1 above)
Comment (katjuell) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md:159]: Tagging @pdeslaur to see if he has thoughts — I don't have a strong opinion either way!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md [hunk 2]
```diff
@@ -25,28 +25,25 @@ To apply a policy to a cluster, you can use the `chainctl` command. Be sure to r
...
 ```
 
-Alternately, you can follow our guide on [How to Create Policies in the Chainguard Enforce UI](how-to-create-policies-chainguard-enforce-ui).
+Alternatively, you can follow our guide on [How to Create Policies in the Chainguard Enforce UI](how-to-create-policies-chainguard-enforce-ui).
 
 ## Policy enforcing signed containers from Chainguard Images
 
 ```
-apiVersion: policy.sigstore.dev/v1beta1
+apiVersion: policy.sigstore.dev/v1alpha1
```
[the remark this answers is not in the document]
Comment (katjuell) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md:33]: I'll open a PR to update this on our side as well

**Pull request merged**